ingest/validation/validate_metadata.py

```diff
@@ -24,7 +24,12 @@
 import jsonschema
 
 sys.path.append('..')
-from cell_metadata import CellMetadata
+try:
+    # Used when importing internally and in tests
+    from cell_metadata import CellMetadata
+except ImportError:
+    # Used when importing as external package, e.g. imports in single_cell_portal code
+    from ..cell_metadata import CellMetadata
```

Comment:
is this just unfamiliar syntax to me or should there be a slash in ../cell_metadata?

Reply:
Python relative path syntax differs from that of POSIX.  It seemed odd to me at first, but the lack of a slash here is standard; see e.g. https://stackoverflow.com/a/1054281.